LibScrollingOptimization/SmoothScrollingBehavior.cs

```diff
@@ -0,0 +1,468 @@
+		/// </summary>
+		public static readonly DependencyProperty VerticalOffsetTargetProperty =
+			VerticalOffsetTargetPropertyKey.DependencyProperty;
+
+
+
+		/// <summary>
+		/// Get value of ScrollWithWheelDelta property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <returns></returns>
+		public static bool GetScrollWithWheelDelta(DependencyObject obj)
+		{
+			return (bool)obj.GetValue(ScrollWithWheelDeltaProperty);
+		}
+
+		/// <summary>
+		/// Set value of ScrollWithWheelDelta property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <param name="value"></param>
+		public static void SetScrollWithWheelDelta(DependencyObject obj, bool value)
+		{
+			obj.SetValue(ScrollWithWheelDeltaProperty, value);
+		}
+
+		/// <summary>
+		/// Get value of EnableScrollingAnimation property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <returns></returns>
+		public static bool GetEnableScrollingAnimation(DependencyObject obj)
+		{
+			return (bool)obj.GetValue(EnableScrollingAnimationProperty);
+		}
+
+		/// <summary>
+		/// Set value of EnableScrollingAnimation property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <param name="value"></param>
+		public static void SetEnableScrollingAnimation(DependencyObject obj, bool value)
+		{
+			obj.SetValue(EnableScrollingAnimationProperty, value);
+		}
+
+		/// <summary>
+		/// Get value of ScrollingAnimationDuration property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <returns></returns>
+		public static Duration GetScrollingAnimationDuration(DependencyObject obj)
+		{
+			return (Duration)obj.GetValue(ScrollingAnimationDurationProperty);
+		}
+
+		/// <summary>
+		/// Set value of ScrollingAnimationDuration property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <param name="value"></param>
+		public static void SetScrollingAnimationDuration(DependencyObject obj, Duration value)
+		{
+			obj.SetValue(ScrollingAnimationDurationProperty, value);
+		}
+
+
+		/// <summary>
+		/// Set value of AlwaysHandleMouseWheelScrolling property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <returns></returns>
+		public static bool GetAlwaysHandleMouseWheelScrolling(DependencyObject obj)
+		{
+			return (bool)obj.GetValue(AlwaysHandleMouseWheelScrollingProperty);
+		}
+
+		/// <summary>
+		/// Get value of AlwaysHandleMouseWheelScrolling property
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <param name="value"></param>
+		public static void SetAlwaysHandleMouseWheelScrolling(DependencyObject obj, bool value)
+		{
+			obj.SetValue(AlwaysHandleMouseWheelScrollingProperty, value);
+		}
+
+
+		/// <summary>
+		/// The DependencyProperty of <see cref="ScrollWithWheelDelta"/> property.
+		/// </summary>
+		public static readonly DependencyProperty ScrollWithWheelDeltaProperty =
+			DependencyProperty.RegisterAttached(nameof(ScrollWithWheelDelta), typeof(bool), typeof(SmoothScrollingBehavior),
```

Comment:
We don't need attached properties with inheritance, since the properties now can be directly set on the behavior.